mantidproject/mantid
# Fix error reporter not launching on MantidPlot

**Description of work.**
This PR fixes the error reporter not launching for MantidPlot due to the launch script pointing to the wrong path.



**To test:**

1. Code review
2. Test the package builds for OSX and Windows. 
3. Open Mantid plot
4. Run the segfault algorithm and check that the error reporter correctly appears.



Fixes #29329 . 



This does not require release notes as it was an issue introduced this sprint.


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
